addons/binding/org.openhab.binding.tankerkoenig/src/main/java/org/openhab/binding/tankerkoenig/internal/serializer/CustomTankerkoenigListResultDeserializer.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * Copyright (c) 2014-2017 by the respective copyright holders.
```

Comment:
It should be 2010-2017, can you please replace them manually if the script did not do it correctly, just search replace for 
"Copyright (c) 2014-2017"  => "Copyright (c) 2010-2017"